doc/releasenotes/16_0_0_changelog.md

```diff
@@ -0,0 +1,403 @@
+ * Remove replication manager and run VTOrc in all e2e tests [#12149](https://github.com/vitessio/vitess/pull/12149) 
+#### General
+ * Improve Codeowners File [#11428](https://github.com/vitessio/vitess/pull/11428)
+ * Remove example script that caused some confusion [#11529](https://github.com/vitessio/vitess/pull/11529)
+ * Remove unused ioutil2 code [#11661](https://github.com/vitessio/vitess/pull/11661)
+ * Fix some linter errors [#11773](https://github.com/vitessio/vitess/pull/11773)
+ * Remove Deprecated flags, code and stats. [#12083](https://github.com/vitessio/vitess/pull/12083) 
+#### Governance
+ * Correct minor inaccuracies in governing docs [#11933](https://github.com/vitessio/vitess/pull/11933) 
+#### Online DDL
+ * [cleanup] Explicitly include DDLStrategySetting in the sizegen target [#11857](https://github.com/vitessio/vitess/pull/11857)
+ * OnlineDDL: avoid schema_migrations AUTO_INCREMENT gaps by pre-checking for existing migration [#12169](https://github.com/vitessio/vitess/pull/12169) 
+#### Query Serving
+ * [gen4 planner] Operator refactoring [#11498](https://github.com/vitessio/vitess/pull/11498)
+ * [gen4]: small refactoring around Compact [#11537](https://github.com/vitessio/vitess/pull/11537)
+ * change CreatePhysicalOperator to use the rewriteBottomUp() functionality [#11542](https://github.com/vitessio/vitess/pull/11542)
+ * [refactor planner] Columns and predicates on operators [#11606](https://github.com/vitessio/vitess/pull/11606)
+ * Move initialization of metrics to be static [#11608](https://github.com/vitessio/vitess/pull/11608)
+ * planner operators refactoring [#11680](https://github.com/vitessio/vitess/pull/11680)
+ * sqlparser: new Equality API [#11906](https://github.com/vitessio/vitess/pull/11906)
+ * sqlparser: `QueryMatchesTemplates` uses canonical string [#11990](https://github.com/vitessio/vitess/pull/11990)
+ * Move more rewriting to SafeRewrite [#12063](https://github.com/vitessio/vitess/pull/12063)
+ * store transaction isolation level in upper case [#12099](https://github.com/vitessio/vitess/pull/12099)
+ * Generating copy-on-rewrite logic [#12135](https://github.com/vitessio/vitess/pull/12135)
+ * Clean up ColumnType uses [#12139](https://github.com/vitessio/vitess/pull/12139) 
+#### TabletManager
+ * Table GC: rely on tm state to determine operation mode [#11972](https://github.com/vitessio/vitess/pull/11972)
+ * Mark VReplicationExec Client Command as Deprecated [#12070](https://github.com/vitessio/vitess/pull/12070) 
+#### VReplication
+ * Leverage pFlag's Changed function to detect user specified flag [#11677](https://github.com/vitessio/vitess/pull/11677)
+ * VReplication: Remove Deprecated V1 Client Commands [#11705](https://github.com/vitessio/vitess/pull/11705) 
+#### VTAdmin
+ * move react-scripts to dev dependencies [#11767](https://github.com/vitessio/vitess/pull/11767) 
+#### web UI
```

Comment:
Web UI. IMO, it probably also makes sense to use `Admin Web UI` to make this a bit more specific (in all locations).